src/main/java/org/ibp/api/rest/keysequenceregister/KeySequenceResource.java

```diff
@@ -0,0 +1,38 @@
+package org.ibp.api.rest.keysequenceregister;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.ibp.api.domain.keysequenceregister.KeySequenceRegisterDeleteResponse;
+import org.ibp.api.java.keysequenceregister.KeySequenceRegisterService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.annotation.Resource;
+import java.util.List;
+
+@Api(value = "Key Sequence Register Services")
+@Controller
+public class KeySequenceResource {
```

Comment:
Logged https://ibplatform.atlassian.net/browse/IBP-4391 to create unit test for this class